### PR TITLE
DM-45939: Use std namespace for cstdint types

### PR DESCRIPTION
**Pull request by mwittgen**
## Checklist

- [X] ran Jenkins
- [ ] added a release note for user-visible changes to `doc/changes`

### CODE REVIEW

**Comment by timj** [comment]
I've added python 3.13 to the build so we get a wheel next time (which will speed up the other 3.13 builds).

**Comment by roceb** [comment]
Many of these files do not include <cstdint>. Can you add them for clarity.

**Comment by mwittgen** [comment]
I'll skip adding `cstdint` in the derived class definitions. I would add them in direct dependencies.

File: include/lsst/sphgeom/Chunker.h [hunk 1]
```diff
@@ -48,8 +48,8 @@ namespace sphgeom {
 ///
 /// TODO(smm): implement a more memory efficient representation than this.
 struct SubChunks {
-    int32_t chunkId;
-    std::vector<int32_t> subChunkIds;
+    std::int32_t chunkId;
```
Comment (timj) [inline include/lsst/sphgeom/Chunker.h:51]: Maybe this file should be including `<cstdint>` and not `<stdint.h>`? (not sure if there are other cases of this).

Comment (timj) [reply include/lsst/sphgeom/Chunker.h:52]:
```$ rg stdint.h sphgeom
sphgeom/include/lsst/sphgeom/BigInteger.h
36:#include <stdint.h>

sphgeom/include/lsst/sphgeom/Chunker.h
37:#include <stdint.h>

sphgeom/tests/test.h
36:#include <stdint.h>```

**Review by roceb — APPROVED** [review]

**Pull request merged**